src/JSONLD/JSONLD.jl

```diff
@@ -0,0 +1,80 @@
+abstract type JSONLD <: DataRepo
+end
+
+export JSONLD_Web, JSONLD_DOI
+
+include("JSONLD_Web.jl")
+include("JSONLD_DOI.jl")
+
+function description(repo::JSONLD, mainpage)
+    desc = filter_html(handle_keys(mainpage, "description"))
+    authors = get_authors(repo, mainpage)
+    author = format_authors(authors)
+    license = get_license(mainpage)
+    date = get_dates(repo, mainpage)
+
+    """
+    Author: $(author)
+    License: $(license)
+    Date: $(date)
+
+    $(desc)
+    """
+end
+
+function get_authors(repo::JSONLD, mainpage)
+    authors = handle_keys(mainpage, "author", "creator")
+    if authors isa Vector
+        return collect(skipmissing(handle_keys.(authors, "name")))
+    elseif authors isa Dict
+        return [handle_keys(authors, "name")]
+    else
+        return []
```

Comment:
maybe add an `@assert(authors==nothing` 
if that is what you are expecting here.

Reply:
Didn't quite understand this

Reply:
Under what circumstances does the else trigger?